src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -1393,34 +1401,33 @@ bool SystemDictionaryShared::should_be_excluded(InstanceKlass* k) {
   }
 
   if (k->has_old_class_version() && k->is_linked()) {
-    warn_excluded(k, "Old class has been linked");
-    return true;
+    return warn_excluded(k, "Old class has been linked");
   }
 
-  InstanceKlass* super = k->java_super();
-  if (super != NULL && should_be_excluded(super)) {
+  if (k->is_hidden() && !is_registered_lambda_proxy_class(k)) {
     ResourceMark rm;
-    log_warning(cds)("Skipping %s: super class %s is excluded", k->name()->as_C_string(), super->name()->as_C_string());
+    log_debug(cds)("Skipping %s: %s", k->name()->as_C_string(), "Hidden class");
```

Comment:
You don't need to feed "Hidden class" to a %s.

Reply:
Fixed.